yt_dlp/extractor/facebook.py

```diff
@@ -481,6 +481,29 @@ def _extract_from_url(self, url, video_id):
         webpage = self._download_webpage(
             url.replace('://m.facebook.com/', '://www.facebook.com/'), video_id)
 
+        def extract_follower_count(webpage):
+            post_data = [self._parse_json(j, video_id, fatal=False) for j in re.findall(
+                r'data-sjs>({.*?ScheduledServerJS.*?})</script>', webpage)]
+            post = traverse_obj(post_data, (
+                ..., 'require', ..., ..., ..., '__bbox', 'require', ..., ..., ..., '__bbox', 'result', 'data'), expected_type=dict) or []
+
+            followers = get_first(post, ('user', 'profile_header_renderer', 'user', 'profile_social_context', 'content', ..., 'text',
+                                         lambda k, v: k == 'text' and isinstance(v, str) and v.endswith('followers'))) or None
+            if not isinstance(followers, str):
+                return None
+
+            matches = re.search(r"(\d+)([K|M])?", followers)
+            if matches is None:
+                return None
+
+            count = int(matches[1])
+            unit = matches[2]
+            if unit == "K":
+                count *= 1_000
+            elif unit == "M":
+                count *= 1_000_000
+            return count
+
```

Comment:
don't need all this. `post_data` and `post` are just duplicated code from `extract_metadata`, and everything below `followers =` is just reinventing the `utils.parse_count` wheel

```suggestion

```